reframe/core/schedulers/slurm.py

```diff
@@ -31,6 +31,9 @@ class SlurmJobState(sched.JobState):
 SLURM_JOB_SUSPENDED   = SlurmJobState('SUSPENDED')
 SLURM_JOB_TIMEOUT     = SlurmJobState('TIMEOUT')
 
+# Number of _update_state calls per which _cancel_if_blocked is called
+SACCT_SQUEUE_RATIO = 20
```

Comment:
I think this constant should be defined inside the `SlurmJob` class as class attribute, since it is very specific to this implementation.

Reply:
@victorusu Should we start with 10 and see if ops are happy?